src/ui/server/controllers/insurance/policy/other-company-details/validation/rules/company-name/index.ts

```diff
@@ -0,0 +1,25 @@
+import { POLICY as POLICY_FIELD_IDS } from '../../../../../../../constants/field-ids/insurance/policy';
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+import { POLICY_FIELDS } from '../../../../../../../content-strings/fields/insurance/policy';
+import providedAndMaxLength from '../../../../../../../shared-validation/provided-and-max-length';
+import { RequestBody } from '../../../../../../../../types';
+
+const {
+  REQUESTED_JOINTLY_INSURED_PARTY: { COMPANY_NAME: FIELD_ID },
+} = POLICY_FIELD_IDS;
+
+const {
+  REQUESTED_JOINTLY_INSURED_PARTY: { [FIELD_ID]: ERROR_MESSAGES_OBJECT },
+} = ERROR_MESSAGES.INSURANCE.POLICY;
+
+const MAXIMUM = Number(POLICY_FIELDS.REQUESTED_JOINTLY_INSURED_PARTY[FIELD_ID].MAXIMUM);
+
+/**
+ * validate the "company name" in other company details response body
+ * @param {Express.Request.body} responseBody containing an object with the company details response
+ * @param {Object} errors errorList
+ * @returns {object} object containing errors or blank object
```

Comment:
Capital O in object

Reply:
I noticed we have this in many places - will fix them all, but have fixed this for now